packages/docs/docs/_changelog/1.13.x.md

```diff
@@ -0,0 +1,27 @@
+---
+version: "1.13.x"
+date: "2025-11-07"
+title: "Physical Android Devices, Apollo Client DevTools, Radon Pro & Team Plans"
+---
+
+- [**Physical Android Devices**](/docs/guides/physical-devices): Connect, build & run your application on physical Android devices.
+
+<video autoPlay loop width="700" controls className="shadow-image changelog-item">
+  <source src="/video/radon_physical_devices.mp4" type="video/mp4" />
+</video>
+
+- [**Apollo Client DevTools**](/docs/features/dev-tools#apollo-client): Inspect Apollo Client logs via built-in Apollo Client DevTools panel.
+
+<img
+  width="600"
+  src="/img/docs/ide_devtools_apollo_client.png"
+  className="shadow-image changelog-item"
+/>
+
+- **`view_component_tree` tool**: Radon AI can now request to view the currently mounted React component tree of your application.
+
+- Lots of improvements to the Network Inspector, React Query DevTools, and Radon AI.
```

Comment:
```suggestion
- Lots of improvements to the Network Inspector, Redux DevTools, and Radon AI.
```
we've had a full reimplementation of Redux, and small fixes to React Query (related to plugin detection), so I assume this is what you meant.

Reply:
m'kay